c/src/eph_decode.c

```diff
@@ -98,7 +97,6 @@ rtcm3_rc rtcm3_decode_gps_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
  * \param RTCM message struct
  * \return  - RC_OK : Success
  *          - RC_MESSAGE_TYPE_MISMATCH : Message type mismatch
- *          - RC_INVALID_MESSAGE : Cell mask too large or invalid TOW
  */
 rtcm3_rc rtcm3_decode_glo_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
```

Comment:
can we add 
```
  msg_eph->constellation = RTCM_CONSTELLATION_GLO;
```
here too?

Reply:
and in general also something like
```suggestion
rtcm3_rc rtcm3_decode_glo_eph(const uint8_t buff[], rtcm_msg_eph *msg_eph) {
  assert(msg_eph);
  memset(msg_eph, 0, sizeof(rtcm_msg_eph));
```